tensorflow_datasets/testing/dummy_dataset/dummy_dataset_checksums_test.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+from tensorflow_datasets import testing
+from dummy_dataset_checksums import DummyDatasetChecksums
+
+
+class DummyDatasetChecksumTest(testing.DatasetBuilderTestCase):
+  DATASET_CLASS = DummyDatasetChecksums
+  EXAMPLE_DIR = os.path.join(DummyDatasetChecksums.code_dir, "fake_examples")
```

Comment:
As for checksums, this should also be automatically detected. The user shouldn't have to manually set this.